docs/src/installation.md

```diff
@@ -1,233 +0,0 @@
----
```

Comment:
How come this is unused? IIUC https://playwright.dev/python/docs/next/installation/ will break after this PR.